keyboards/fjlabs/fe87/keyboard.json

```
@@ -0,0 +1,132 @@
+{
+    "keyboard_name": "Fe87",
+    "manufacturer": "FJLaboratories",
+    "url": "http://www.fjlaboratories.com/",
+    "maintainer": "mrnoisytiger",
+    "usb": {
+        "vid": "0x7074",
+        "pid": "0x2642",
+        "device_version": "0.0.1"
+    },
+    "matrix_pins": {
+        "cols": ["F7", "F6", "F5", "F4", "F3", "F2", "F1", "E2", "C7", "C6", "C5", "C4", "C3", "C2", "C1", "D6", "D7"],
+        "rows": ["B3", "B4", "B5", "C0", "E1", "E0"]
+    },
+    "diode_direction": "COL2ROW",
+    "processor": "at90usb646",
+    "bootloader": "atmel-dfu",
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "nkro": true
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "community_layouts": ["tkl_ansi"],
+    "layouts": {
+        "LAYOUT_tkl_ansi": {
```

Without an image in readme.md, am assuming this is a PCB with more than one layout thus LAYOUT_all by PR Checklist is correct here.
```suggestion
    "layouts": {
        "LAYOUT_all": {
```
Assuming further it's compatible with community layouts:
- tkl_ansi
- tkl_ansi_split_bs_rshift
- tkl_iso
- tkl_iso_split_bs_rshift